community/modules/scripts/pbspro-preinstall/README.md

```diff
@@ -0,0 +1,141 @@
+## Description
+
+This module uploads PBS Pro RPM packages and, optionally, a license file to
+Google Cloud Storage. This enables machines in a PBS cluster to rapidly
+download and install PBS at boot or during the building of an image.
+
+### Example
+
+The following code snippet uses this module to
```

Comment:
Missing the end of the sentence.

Reply:
Addressed on HEAD of PR. I think the text went missing during some branch re-arrangement.